### PR TITLE
docs: include branch setup for building snaps in README

### PR DESCRIPTION
**Pull request by d-loose**
Adds a section to the README that explains multiple versions for
multiple snaps can be build from this monorepo.

UDENG-4451

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -148,11 +148,27 @@ represents Brazilian Portuguese.
 
 ## Repository Structure
 
+### Branches and Snap Packages
+
+Active development for all snaps built from this monorepo takes place on the
+`main` branch. To continue supporting Ubuntu releases with bug fixes dedicated
```
Comment (matthew-hagemann) [inline README.md:154]: I'd add a comma after "fixes" for sentence flow

File: README.md [hunk 2]
```diff
@@ -148,11 +148,27 @@ represents Brazilian Portuguese.
...
+branches (e.g. `ubuntu/24.04`) are created once development on features for an
+upcomiingrelease begins.
```
Comment (matthew-hagemann) [inline README.md:156]: upcoming release

**Review by matthew-hagemann — APPROVED** [review]
LGTM! Two small requests.

File: README.md (hunk 2 above)
Comment (d-loose) [reply README.md:156]: I thought I spell-checked this :D thanks

**Pull request merged**